new_platforms/vscode-extension/assets/standaloneSolutionTemplateFolder/host/main.c

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include <stdio.h>
+
+#include <openenclave/host.h>
+
+int open_enclave();
+int close_enclave();
+int call_enclave(char *input_msg, char *enclave_msg, unsigned int enclave_msg_size);
+
+int main(int argc, const char* argv[])
+{
+    int result = open_enclave();
+    if (result != 0)
+    {
+        return result;
+    }
+    char* enclaveMessage = (char*)malloc(512 * sizeof(char));
+    result = call_enclave("\"ProcessInEnclave\"", enclaveMessage, 512);
```

Comment:
Maybe ok for this checkin, but what is the string "ProcessInEnclave" used for?   If it's only used in output read by humans then there's no reason to omit spaces inside it.  If it's machine processable to be matched with something then it shouldn't be a string literal here, it would be a define or something that could be used by the other location too.   So I'm confused by this code.

Reply:
in essence, whatever is passed in here will be wrapped in a layer of json by the enclave.